docs/redirects.txt

```diff
@@ -0,0 +1,93 @@
+# Client-side page redirects
+# ---------------------------
+# Comments start with a single hash (#)
+# Each redirect appears on its own line in the format:
+#
+# redirect/from/ redirect/to/
+#
+# Paths are relative to the root of the `docs` directory
+# The "from" path must be a file that *does not* exist
+# The "to" path must be a file that *does* exist
+# The separator between "from" and "to" paths is a single space (not a tab)
+#
+# In this implementation:
+#   Requests to help.launchpad.net are forwaded to the same path at https://documentation.ubuntu.com/launchpad/user
```

Comment:
If I recall correctly, IS applied a DNS level redirect, which means, it cannot redirect to subdirectories or URL that contain paths. Has this been changed? @lgp171188